sycl/include/CL/sycl/detail/pi.h

```diff
@@ -111,6 +111,8 @@ typedef enum {
   PI_INVALID_IMAGE_FORMAT_DESCRIPTOR = CL_INVALID_IMAGE_FORMAT_DESCRIPTOR,
   PI_IMAGE_FORMAT_NOT_SUPPORTED = CL_IMAGE_FORMAT_NOT_SUPPORTED,
   PI_MEM_OBJECT_ALLOCATION_FAILURE = CL_MEM_OBJECT_ALLOCATION_FAILURE,
+  PI_FALLBACK_FAILURE = -997,
+  PI_FALLBACK_SUCCESS = -998,
```

Comment:
Why do you need 2 new values and not just 1 like PI_FUNCTION_ADDRESS_IS_NOT_AVAILABLE (meaning function s found but it's address cannot be obtained)? Also please document this in the source code comment.

Reply:
Two was not needed I have implemented `PI_FUNCTION_ADDRESS_IS_NOT_AVAILABLE` and am returning `PI_INVALID_KERNEL_NAME` if the kernel name does not exist.